src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/ObjectSequence.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Numerics;
+using System.Runtime.InteropServices;
+
+namespace System.Diagnostics.Metrics
+{
+    internal partial struct ObjectSequence1 : IEquatable<ObjectSequence1>, IObjectSequence
+    {
+        public object? Value1;
+
+        public ObjectSequence1(object? value1)
+        {
+            Value1 = value1;
+        }
+
+        public override int GetHashCode() => Value1?.GetHashCode() ?? 0;
```

Comment:
Is Value1 ever written outside of the ctor?  If no, it should be readonly.  If yes, does any code depend on this GetHashCode being stable, e.g. are these ever put into a dictionary?

Reply:
Is Value1 ever written outside of the ctor? Nope